### PR TITLE
chore: Add the `steps` property

### PR DESCRIPTION
**Pull request by blendthink**
## Description

This pull request addresses an issue in the official Melos schema configuration for the `melos.yaml` file. Specifically, it adds the missing schema definition for the `steps` field.

### Problem

The current official schema lacks the definition for the `steps` field in `melos.yaml`. This omission causes validation errors when users attempt to use the `steps` field in their `melos.yaml` files.

image

### Solution

Add the `steps` property to the `script` definition in the `$defs.script` section of `melos.yaml.schema.json`.

image

### Validation

The changes have been validated in the repository at https://github.com/blendthink/melos-yaml-scheme.

## Type of Change

- [ ] ✨ `feat` -- New feature (non-breaking change which adds functionality)
- [x] 🛠️ `fix` -- Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ `!` -- Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 `refactor` -- Code refactor
- [ ] ✅ `ci` -- Build configuration change
- [x] 📝 `docs` -- Documentation
- [x] 🗑️ `chore` -- Chore

### CODE REVIEW

**Review by spydon — APPROVED** [review]
Lgtm, thanks for your contribution! :)

**Comment by blendthink** [comment]
@spydon 
I submitted a pull request, but some of the CI checks failed.
https://github.com/invertase/melos/actions/runs/11529828753/job/32099370944

It seems that this issue has occurred before as well, and it appears to be a randomly occurring problem.
https://github.com/invertase/melos/actions/runs/11222568392/job/31195282729

I attempted to rerun the CI by creating and pushing an empty commit, but I forgot that it requires maintainer approval.
I apologize for the inconvenience, but could you please help by rerunning the CI?

**Comment by blendthink** [comment]
Thank you for rerunning the CI!

**Pull request merged**
